store/models.py

```diff
@@ -30,3 +32,9 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+class Rate(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rater = models.ForeignKey(User, related_name='rater',null=True,on_delete=models.SET_NULL)
+    rating = models.FloatField(validators=[MinValueValidator(1), MaxValueValidator(10)],default=10.0)
```

Comment:
Good use of validators here.
However, the rater can't be null and the rating shall be an integer - please read proper instructions.

Also, you could have also used `unique_together` META option here.